site/content/examples/05-bindings/02-checkbox-inputs/App.svelte

```diff
@@ -10,7 +10,7 @@
 {#if yes}
 	<p>Thank you. We will bombard your inbox and sell your personal details.</p>
 {:else}
-	<p>You must opt in to continue. If you're not paying, you're the product.</p>
+	<p>You must opt-in to continue. If you're not paying, you're the product.</p>
 {/if}
 
 <button disabled={!yes}>
```

Comment:
I can't hang comments on file renames, but the next file is being renamed because `automaticly` isn't the proper spelling...